config/routes.rb

```diff
@@ -145,7 +145,7 @@ def scoped_navigation_routes(context, navigation, as_redirects: false)
     get "/users/invitations" => "invitations#index", as: :invitations
```

Comment:
Can we take this opportunity to namespace these user routes under hub too?

Reply:
We can probably move most if not all of them. It might be helpful to separate that step out, because for the devise routes it would take a bit more work.

Reply:
Sure, I'll start on that. I do think it would be good to merge this and do a separate PR unless you think there's something in the current commit that would cause a problem.